sg_bookmarks.user.js

```diff
@@ -24,18 +24,23 @@
 
 //GA Entered classes
 GM_addStyle(".nav__relative-dropdown.___mh_bookmark_outer_container {z-index:1000;}");
-GM_addStyle(".nav__row.__mh_bookmark_item.__mh_state_entered {background-image: linear-gradient(#f6f6ec 0%, #e9e9ca 100%);\
-background-image: -moz-linear-gradient(#f6f6ec 0%, #e9e9ca 100%);\
-background-image: -webkit-linear-gradient(#f6f6ec 0%, #e9e9ca 100%);}");
+GM_addStyle(".nav__row.__mh_bookmark_item.__mh_state_entered *{opacity:0.7;}");
+GM_addStyle(".nav__row.__mh_bookmark_item.__mh_state_entered.mid_train .train_tracks {opacity: 0.17; padding: 0px 4px 0px 4px}");
+//GM_addStyle(".nav__row.__mh_bookmark_item.__mh_state_entered {background-image: linear-gradient(#f6f6ec 0%, #e9e9ca 100%);background-image: -moz-linear-gradient(#f6f6ec 0%, #e9e9ca 100%);background-image: -webkit-linear-gradient(#f6f6ec 0%, #e9e9ca 100%);}");
 //GA Owned classes
 GM_addStyle(".nav__row.__mh_bookmark_item.__mh_state_owned {background-image: linear-gradient(#f7edf1 0%, #e6d9de 100%);\
 background-image: -moz-linear-gradient(#f7edf1 0%, #e6d9de 100%);\
 background-image: -webkit-linear-gradient(#f7edf1 0%, #e6d9de 100%);}");
+GM_addStyle(".nav__row.__mh_bookmark_item.__mh_state_owned::hover { color: #111; }");
 //Train classes
 //GM_addStyle(".nav__row.__mh_bookmark_item{padding-bottom:10px}");
-GM_addStyle(".nav__row.__mh_bookmark_item.mid_train{height: 20px;overflow:hidden;margin: 1px 1px 1px 10px;}");
+//GM_addStyle(".nav__row.__mh_bookmark_item{border-top:1px solid #DDD;}");
+//GM_addStyle(".nav__row.__mh_bookmark_item{z-index:9;position:relative;box-shadow: 0px 1px 5px -1px #ccc;}");
+GM_addStyle(".nav__row.__mh_bookmark_item.mid_train{box-shadow: inset 0 0 10px -9px #000000;height: 20px;overflow:hidden;margin: 0px 0px 0px 2px;}");
 GM_addStyle(".nav__row.__mh_bookmark_item.mid_train .nav__row__summary__name{white-space: nowrap;}");
 GM_addStyle(".nav__row.__mh_bookmark_item.mid_train .nav__row__summary__description{display:none;}");
+GM_addStyle(".nav__row.__mh_bookmark_item.mid_train .train_tracks {opacity: 0.3; padding: 0px 4px 0px 4px}");
```

Comment:
I'd like to decrease the opacity a little bit more. Padding changes are for adjusting the padding from my first comment.
{opacity: 0.2; padding: 0px 6px 0px 3px}